tuya.cpp

```diff
@@ -353,6 +354,28 @@ void DeRestPluginPrivate::handleTuyaClusterIndication(const deCONZ::ApsDataIndic
             }
         }
 
+        //Some device have sensor created on other endpoint and other cluster but are using the endpoint 0x01 and the cluster 0xEF00
+        if (sensorNode && R_GetProductId(sensorNode) == QLatin1String("Tuya_SEN Multi-sensor"))
+        {
+            switch (dp)
+            {
+                //temperature
+                case 0x026B:
```

Comment:
Good job on the DP identifier constants, for the remaining magic values it would be nice to have a separate PR to replace them with constants too.

Reply:
Yes it s on the TODOLIST, value are already in the tuya.h file, and used in the rest of code.

I need to rewrite the complete tuya.cpp code to separate, the dp_type and the dp_value, but the code still too much, for exemple https://github.com/Smanar/deconz-rest-plugin/commit/2e40d7f4b823908ed72f6f47a603b79346e6c220.